### PR TITLE
Only change rails configuration if Rails.application exits

### PR DESCRIPTION
**Pull request by tmuntaner**
`rmt-cli` is broken because `Rails.application` is nil on cli commands.

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: config/initializers/fast_gettext.rb [hunk 1]
```diff
@@ -6,4 +6,4 @@
 FastGettext.text_domain = FastGettext.default_text_domain = 'rmt'
 
 # enable comments in .po/.pot files prefixed with i18n tag (see rxgettext -h)
-Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=i18n]
+Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=i18n] unless Rails.application.nil?
```
Comment (ikapelyukhin) [inline config/initializers/fast_gettext.rb]: `if Rails.application`.

I'd still move it into a separate initializer. Or better yet not use Rails initializer in CLI script (it's rather confusing that it's being used there -- why wouldn't `Rails.application` be defined in Rails initializer :smile:).

Comment (tmuntaner) [reply config/initializers/fast_gettext.rb]: I think the problem is that this initializer is used for both rails and `rmt-cli`. I guess we would need a new initializers directory to help prevent this happening again.

Comment (ikapelyukhin) [reply config/initializers/fast_gettext.rb]: Just put FastGettext stuff into `lib`, and then require it from Rails/CLI?

Comment (tmuntaner) [reply config/initializers/fast_gettext.rb]: I will try that

File: config/initializers/fast_gettext.rb [hunk 2]
```diff
@@ -1,9 +1,4 @@
...
-rmt_path = File.expand_path('../../', __dir__)
-FastGettext.add_text_domain('rmt', path: File.join(rmt_path, 'locale'), type: :po, ignore_fuzzy: true, report_warning: false)
-FastGettext.default_locale = :en
-FastGettext.available_locales = FastGettext.default_available_locales = %w[en de fr ja es pt_BR zh_CN]
-FastGettext.text_domain = FastGettext.default_text_domain = 'rmt'
+require 'initializers/rmt_fast_gettext'
 
 # enable comments in .po/.pot files prefixed with i18n tag (see rxgettext -h)
-Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=i18n]
+Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=i18n] if Rails.application
```
Comment (ikapelyukhin) [inline config/initializers/fast_gettext.rb]: `if Rails.application` isn't needed anymore.

Comment (tmuntaner) [reply config/initializers/fast_gettext.rb]: I forgot to remove that one ^_^

**Comment by tmuntaner** [comment]
@ikapelyukhin thanks

**Pull request merged**